src/main/java/com/atlassian/jira/cloud/jenkins/listeners/AutoBuildsListener.java

```diff
@@ -191,7 +192,7 @@ private boolean canDetermineFinalResultOfEndNode() {
             return state != State.IN_PROGRESS;
         } catch (final IOException e) {
             final String message = "cannot determine status: " + e.getMessage();
-            pipelineLogger.println("[WARN] " + message);
```

Comment:
This is how pipeline-logging was done before: just a `println` on a given `PrintStream`. The new `PipelineLogger` class encapsulates this.